polaris-react/src/components/IndexTable/IndexTable.stories.tsx

```diff
@@ -107,6 +108,7 @@ export function Default() {
   return (
     <LegacyCard>
       <IndexTable
+        condensed={useBreakpoints().smDown}
```

Comment:
Flush story looks broken when condensed. Table Cells don't have proper paddings.
<img width="1435" alt="image" src="https://github.com/Shopify/polaris/assets/104942025/c9d1de6e-9abc-43b9-a023-69b36372306c">

Reply:
This is how the flush prop is supposed to work. It's not pretty in this example but it shows what the prop does